webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -164,3 +177,21 @@ const objectDeepTrim = (obj, trigger) => {
   });
   return copy;
 };
+
+export const updateDisability = (isDisabled, id) => ({
+  type: AUTO_COMPLETE_DISABLED_CHANGE,
+  payload: {
+    isDisabled,
+    id,
+  },
+});
+
+export const updateController = (controller, url, id) => ({
+  type: AUTO_COMPLETE_CONTROLLER_CHANGE,
```

Comment:
It seems like this action is never dispatched anywhere except in `index.js.erb` in the initial page load.  Going back to my questions from above, should this be dispatched when the user changes the Resource Type dropdown?